### DISCUSSION TITLE
Shorten SSH certificates expire date

### DISCUSSION THREAD

**Discussion by lkwjohn** [gravitational/teleport#10784] [Q&A] [answered]
The current SSH certificates expiration date is set with a default 12 hours, is there a way to reduce the SSH certificates expiration date for the user login?

**Comment by scottpgallagher** [comment, marked as answer]
If no TTL is given it will default to 12hours, but the way the TTLs work in roles is the user will be given the least amount of time specified. For example if they have 3 roles assigned to them with different TTLs of 8h, 6h, and 2h, then their TTL will be set to 2h. You can set this in the role as such

  options:
    cert_format: standard
    forward_agent: true
    max_session_ttl: 2h0m0s
